src/cacheServerClient/cacheServerClient.types.ts

```diff
@@ -55,6 +56,7 @@ export interface IOrganization {
   apps?: IApp[];
   roles?: IRole[];
   subOrgs?: IOrganization[];
+  isOwned?: boolean;
```

Comment:
Why is it only nullable in the case of Organization (and not nullable for Role and App)? Is it because it doesn't have meaning in the context of top-level org? If so, maybe add a comment to explain the reason

Reply:
Thanks for noticing. Will fix